docs/ruleset_functions/get_rmr_scc_window_wall_ratios.md

```diff
@@ -1,15 +1,15 @@
 
-## get_building_scc_window_wall_ratios
+## get_rmr_scc_window_wall_ratios
```

Comment:
We have now renamed RMR to RMD. So I suggest using this new abbreviation going forward.